detoks/src/main/java/nl/tudelft/trustchain/detoks/DeToksCommunity.kt

```diff
@@ -99,11 +123,79 @@ class DeToksCommunity(private val context: Context) : Community() {
         }
     }
 
+    override fun walkTo(address: IPv4Address) {
+        super.walkTo(address)
+
+        discoveredAddressesContacted[address] = Date()
```

Comment:
Not sure why an empty date constructor is needed here, since itn not the current datetime it could be added later when used